kubernetes/check.py

```diff
@@ -34,6 +34,8 @@
     'cpu.*.total']
 DEFAULT_COLLECT_EVENTS = False
 DEFAULT_NAMESPACES = ['default']
+DEFAULT_SEND_CPU_AS_CORES = False
```

Comment:
You're right, metadata will be a problem. Still, we can't enable it once and for all because it would break monitors for users. Let us try something with metadata, we'll keep you updated on this PR. cc @xvello